webpack.js

```diff
@@ -1,3 +1,10 @@
 const webpackConfig = require('@nextcloud/webpack-vue-config')
 
+const path = require('path')
+
+webpackConfig.entry = {
+	...webpackConfig.entry,
+	dashboard: path.join(__dirname, 'src', 'init-talk.js'),
```

Comment:
```suggestion
	talk: path.join(__dirname, 'src', 'init-talk.js'),
```

Reply:
I don't see the action after changing this, do you have another idea?